index.bs

```diff
@@ -9733,7 +9733,9 @@ various channel configurations can be achieved.
 	{{AudioWorkletNodeOptions/numberOfOutputs}} = 1
 :: If {{AudioWorkletNodeOptions/outputChannelCount}} is
 	unspecified, the output channel count will match
-	<a>computedNumberOfChannels</a> from the input.
+	<a>computedNumberOfChannels</a> from the input. If
+	{{AudioWorkletNodeOptions/outputChannelCount}} is specified
+	with one value, use the value to configure the output channel.
```

Comment:
I think you also need to say what happens if outputChannelCount is specified with more than one value.

Reply:
No need to do anything here then.

Reply:
I like the suggestion of

> "If {{AudioWorkletNodeOptions/outputChannelCount}} is specified with one value, the output channel count will match the given value"

Reply:
Done.